code/datums/gamemodes/spy_theft.dm

```diff
@@ -577,7 +584,7 @@
 			choice = pick(S)
 			item_existing = locate(choice)
 			var/turf/T = get_turf(item_existing)
-			if (item_existing && T.z == 1)
+			if (item_existing && istype(T) && T.z == 1)
```

Comment:
type should be guaranteed, if you got errors then T was null

Reply:
you're right, `T` was `null`. I changed this from `istype` to `!isnull`.